src/interface/src/app/plan/permissions.ts

```diff
@@ -3,3 +3,10 @@ import { Plan } from '../types';
 export function canViewCollaborators(plan: Plan) {
   return plan.permissions?.includes('view_collaborator');
 }
+
+export function canAddScenario(plan: Plan) {
+  if (!plan.permissions) {
```

Comment:
with the latest cleanup you shouldn't need to check this (permissions is no longer optional)